processor/src/main/java/com/linecorp/decaton/processor/runtime/SubscriptionBuilder.java

```diff
@@ -166,8 +166,15 @@ public ProcessorSubscription build() {
 
         DecatonProcessorSupplier<byte[]> retryProcessorSupplier = null;
         if (retryConfig != null) {
-            Properties producerConfig = Optional.ofNullable(retryConfig.producerConfig())
-                                                .orElseGet(producerConfigSupplier(consumerConfig));
+            Properties producerConfig = new Properties();
+            // In Decaton processor which handles massive traffic, retry tasks could cause
+            // production burst when processes start to fail. (e.g. due to downstream service down)
+            // Since default producer's linger.ms is 0, this could harm Kafka cluster despite we
+            // don't much care about retry task's delivery latency typically. So we set reasonable default here.
+            producerConfig.setProperty(ProducerConfig.LINGER_MS_CONFIG, "100");
```

Comment:
Can you pull this up to somewhere like https://github.com/line/decaton/blob/master/client/src/main/java/com/linecorp/decaton/client/internal/DecatonTaskProducer.java#L42 for ability to easy spot where such defaults are defined.

Reply:
I thought about that, but I think making such high linger as the default is only acceptable for retry producers.

But yeah, defining in-line could make spotting the default difficult.
Let me find better way to define defaults for retry producer.